openfisca_web_api_preview/openAPI.yml

```diff
@@ -24,6 +24,7 @@ tags:
       description: "Variable official documentation"
       url: "http://openfisca.org/doc/variables.html"
   - name: "Calculations"
+  - name: "Docu"
```

Comment:
Missing `mentation` ?

Further you seem to refer to a tag name `Documentation `